dipy/segment/quickbundles.py

```diff
@@ -5,102 +5,110 @@
 
 
 class QuickBundles(object):
-    
-    def __init__(self,tracks,dist_thr=4.,pts=12):
-        """ Highly efficient trajectory clustering 
-        
+
+    def __init__(self, tracks, dist_thr=4., pts=12):
+        """ Highly efficient trajectory clustering
+
         Parameters
         -----------
         tracks : sequence of (N,3) ... (M,3) arrays,
                     trajectories (or tractography or streamlines)
-                    
-        dist_thr : float, 
+
+        dist_thr : float,
                     distance threshold in the space of the tracks
-        pts : int, 
-                number of points for simplifying the tracks 
-                       
+        pts : int,
+                number of points for simplifying the tracks
+
         Methods
         --------
         clustering() returns a dict holding with the clustering result
         virtuals() gives the virtuals (track centroids) of the clusters
-        exemplars() gives the exemplars (track medoids) of the clusters        
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
         Citation
         ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+
+        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
+
         """
-        warn(DeprecationWarning("Class 'dipy.segment.quickbundles.QuickBundles'"
-                                " is deprecated, use module "
-                                "'dipy.segment.clustering.QuickBundles'"
-                                " instead"))
+        warn(
+            DeprecationWarning(
+                "Class 'dipy.segment.quickbundles.QuickBundles'"
+                " is deprecated, use module "
+                "'dipy.segment.clustering.QuickBundles'"
+                " instead"))
 
         self.dist_thr = dist_thr
         self.pts = pts
-        if pts!=None:                        
-            self.tracksd=[downsample(track,self.pts) for track in tracks]
+        if pts is not None:
+            self.tracksd = [downsample(track, self.pts) for track in tracks]
         else:
-            self.tracksd=tracks                    
-        self.clustering=local_skeleton_clustering(self.tracksd, self.dist_thr)
-        self.virts=None
-        self.exemps=None                
-    
+            self.tracksd = tracks
+        self.clustering = local_skeleton_clustering(
+            self.tracksd, self.dist_thr)
+        self.virts = None
+        self.exemps = None
+
     def virtuals(self):
-        if self.virts==None:
-            self.virts=[self.clustering[c]['hidden']/np.float(self.clustering[c]['N']) for c in self.clustering]
-        return self.virts      
+        if self.virts is None:
+            self.virts = [
+                self.clustering[c]['hidden'] /
+                np.float(
+                    self.clustering[c]['N']) for c in self.clustering]
```

Comment:
This is broken into way too many lines. How about the following? 

self.virts = [self.clustering[c]['hidden'] / 
np.float(self.clustering[c]['N']) for c in self.clustering]

With the second line properly indented.